setup.py

```diff
@@ -9,8 +9,8 @@
 with io.open(os.path.join(os.path.dirname(__file__), "README.md"), encoding="utf-8") as f:
     long_description = f.read()
 
-tf_requires = ["tensorflow>=1.0,<2.0"]
-tf_gpu_requires = ["tensorflow-gpu>=1.0,<2.0"]
+tf_requires = ["tensorflow>=1.0,<1.14"]
```

Comment:
I change it because some test related to modelforge and asdf fails. 
We can address it in the following PRs.